### PR TITLE
Update IOR to Systemgruppen

### PR DESCRIPTION
**Pull request by Benjaneb**
## Describe your changes
Mattermost link is changed in this PR, to match when the Mattermost team also gets renamed.

## Checklist before requesting a review

- [x] I have performed a self-review of my changes
  - Instructions for easily viewing your changes locally is available in [the readme](https://github.com/datasektionen/bawang-content?tab=readme-ov-file#k%C3%B6ra-lokalt).
- [x] I have updated both Swedish and English pages (if not motivate why)

### CODE REVIEW

File: namnder/systemgruppen/body_en.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+## The Chapter’s systems
+
+The chapter’s systems are available on our [Github](https://github.com/datasektionen/), if you want access to see or edit something, you can ask Head of IT or come to one of our programming evenings.
+
+### Selection of systems
+
+Here is a list of some systems that previous students have built and that the chapter use extensively:
+- [Cashflow](https://cashflow.datasektionen.se) is probably our most used system. Here you can register expenses that you have made for the chapter and the Treasurer (or some other responsible person) will pay the money to you as soon as possible. Neat, right?
+- [Mentometer](https://mentometer.datasektionen.se) is used to hold certain elections during chapter meetings (SM).
```
Comment (RafDevX) [inline namnder/systemgruppen/body_en.md:13]:
```suggestion
- [Pandora](https://bokning.datasektionen.se) is used to book chapter rooms and equipment.
```

File: namnder/systemgruppen/body_en.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- [aaallt](https://aaallt.datasektionen.se/) may not be our most exciting system, but here you can go to almost every other system the chapter has.
+
+## Join the club
+
+We regularly try to hold programming evenings where everyone is welcome. Usually from 5 pm onwards on Thursdays, usually somewhere in the E-house. You can come here to see what we're doing, develop something for the chapter, hang out or maybe get help with a programming lab. The easiest way to find out when we have events is to ask in our [Mattermost](https://mattermost.datasektionen.se/systemgruppen/channels/town-square)! Hope to see you there!
```
Comment (RafDevX) [inline namnder/systemgruppen/body_en.md:18]:
```suggestion
We regularly try to hold programming evenings where everyone is welcome. Usually from 5 pm onwards on Thursdays, usually somewhere in the E-building. You can come here to see what we're doing, develop something for the chapter, hang out or maybe get help with a programming lab. The easiest way to find out when we have events is to ask in our [Mattermost](https://dsekt.se/ior)! Hope to see you there!
```

File: namnder/systemgruppen/body_en.md
Comment (RafDevX) [inline namnder/systemgruppen/body_en.md:1]: "Systems Group" probably makes more sense than "System Group", no?

File: namnder/systemgruppen/body_sv.md
Comment (RafDevX) [inline namnder/systemgruppen/body_sv.md:1]: Doesn't really match the English version exactly...

File: namnder/systemgruppen/body_sv.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+### Urval av systemen
+
+Här är några system som tidigare studenter har byggt och sen dess använts flitigt:
+- [Cashflow](https://cashflow.datasektionen.se) är nog vårt mest använda system. Här kan du registrera utlägg som du gjort för sektionen och så betalar Kassör (eller annan ansvarig) så snart som möjligt ut pengarna till dig. Smidigt, va??
+- [Damm](https://damm.datasektionen.se) är en samling av bland annat gamla märken och andra historiska saker.
+- [aaallt](https://aaallt.datasektionen.se/) är kanske inte vårt mest spännade system, men här kan du klicka dig vidare till nästan alla andra system sektionen har.
+
+## Engagera dig i nämnden
+
+Vi försöker regelbundet hålla i hackerkvällar där alla sektionsmedlemmar är välkomna. Vanligtvis från klockan 17 och framåt på torsdagar någonstans i E-huset. Hit kan du komma för att hänga med oss, se vad vi håller på med, eller själv utveckla något för sektionen! Om du är intresserad och vill kolla om det är hackerkväll denna vecka, fråga på vår [Mattermost](https://mattermost.datasektionen.se/systemgruppen/channels/town-square)! Hoppas vi ses!
```
Comment (RafDevX) [inline namnder/systemgruppen/body_sv.md:18]:
```suggestion
Vi försöker regelbundet hålla i hackerkvällar där alla sektionsmedlemmar är välkomna. Vanligtvis från klockan 17 och framåt på torsdagar någonstans i E-huset. Hit kan du komma för att hänga med oss, se vad vi håller på med, eller själv utveckla något för sektionen! Om du är intresserad och vill kolla om det är hackerkväll denna vecka, fråga på vår [Mattermost](https://dsekt.se/ior)! Hoppas vi ses!
```

File: namnder/systemgruppen/meta.toml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+sort = 1
+
+[title]
+sv = "Systemgruppen"
+en = "System Group"
```
Comment (RafDevX) [inline namnder/systemgruppen/meta.toml:5]: s

File: namnder/systemgruppen/sidebar_en.md [hunk 5]
```diff
@@ -10,12 +10,13 @@ Mathias Magnusson (previous)</br>
 
 [ior@datasektionen.se](mailto:ior@datasektionen.se)
 
-#### IOR's Mattermost
+#### System Group's Mattermost
 
-<https://mattermost.datasektionen.se/ior/channels/town-square>
+<https://mattermost.datasektionen.se/systemgruppen/channels/town-square>
```
Comment (RafDevX) [inline namnder/systemgruppen/sidebar_en.md:15]:
```suggestion
<https://dsekt.se/ior>
```

File: namnder/systemgruppen/sidebar_sv.md [hunk 6]
```diff
@@ -10,12 +10,13 @@ Mathias Magnusson (förra)</br>
 
 [ior@datasektionen.se](mailto:ior@datasektionen.se)
 
-#### IORs Mattermost
+#### Systemgruppens Mattermost
 
-<https://mattermost.datasektionen.se/ior/channels/town-square>
+<https://mattermost.datasektionen.se/systemgruppen/channels/town-square>
```
Comment (RafDevX) [inline namnder/systemgruppen/sidebar_sv.md:15]:
```suggestion
<https://dsekt.se/ior>
```

File: namnder/systemgruppen/body_en.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
+# System Group
+
+We are happy that you found the System Group's page. We are also known as the _Donkeys_, which is a name that does not describe what we do very well. The System Group is in charge of developing new systems and maintaining the chapter’s current systems. In short, we are a group of students who enjoy programming and learning together. Everyone is welcome here, regardless of whether you have worked as a developer for a few years or are completely new to programming.
```
[the remark this answers is not in the document]
Comment (Benjaneb) [reply namnder/systemgruppen/body_en.md:3]: :donkey:

File: namnder/systemgruppen/sidebar_sv.md (hunk 6 above)
Comment (Benjaneb) [reply namnder/systemgruppen/sidebar_sv.md:15]: Oh yeah I need to update the dsekt link to systemgruppen too. Rename the Mattermost first so I know for sure what to link to.

File: namnder/systemgruppen/body_sv.md
Comment (Benjaneb) [reply namnder/systemgruppen/body_sv.md:1]: Looks like it matches now

**Review by foodelevator — COMMENTED** [review]
Not sure, but I don't think it's possible to change the id/url of a mattermost team?
Also, https://dsekt.se/iormost already exists

**Comment by Herkarl** [comment]
Thought: do we want to rename the mattermost? Could possibly be worth keeping it as a hub for all the parts of the new IOR for the moment? At least until we've tried out stuff a bit and seen what works and not with the new structure.

File: namnder/systemgruppen/sidebar_en.md [hunk 8]
```diff
@@ -6,16 +6,17 @@ Rafael Serra e Oliveira </br>
...
 
-<https://mattermost.datasektionen.se/ior/channels/town-square>
+<https://dsekt.se/iormost>
 
 #### Github
 
 Our [Github](https://dsekt.se/github-link-website) organisation</br>
 
 Do you want access? Ask Head of IT or attend a programming evening!
+
```
Comment (foodelevator) [inline namnder/systemgruppen/sidebar_en.md:22]:
```suggestion
```

File: namnder/systemgruppen/sidebar_sv.md [hunk 9]
```diff
@@ -10,12 +10,13 @@ Mathias Magnusson (förra)</br>
...
 
-<https://mattermost.datasektionen.se/ior/channels/town-square>
+<https://dsekt.se/iormost>
 
 #### Github
 
 [Datasektionens Github](https://dsekt.se/github-link-website)</br>
 
 Vill du ha access? Fråga Systemansvarig eller kom på en hackerkväll!
+
```
Comment (foodelevator) [inline namnder/systemgruppen/sidebar_sv.md:22]:
```suggestion
```

**Review by foodelevator — CHANGES_REQUESTED** [review]

File: namnder/systemgruppen/sidebar_en.md [hunk 10]
```diff
@@ -6,13 +6,13 @@ Rafael Serra e Oliveira </br>
 Mathias Magnusson (previous)</br>
 [d-sys@datasektionen.se](mailto:d-sys@datasektionen.se)
 
-#### System group
+#### Systems group
```
Comment (foodelevator) [inline namnder/systemgruppen/sidebar_en.md:9]:
```suggestion
#### Systems Group
```
när man ändå häller på

**Review by foodelevator — CHANGES_REQUESTED** [review]

**Review by foodelevator — APPROVED** [review]

File: namnder/systemgruppen/sidebar_en.md (hunk 10 above)
Comment (Benjaneb) [reply namnder/systemgruppen/sidebar_en.md]: :pouring_liquid:

**Review by RafDevX — APPROVED** [review]

**Pull request merged**